packages/core/src/app/giftCertificate/AppliedGiftCertificate.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { getStoreConfig } from '@bigcommerce/checkout/test-mocks'
+import { render, screen } from '@bigcommerce/checkout/test-utils';
+
+import { default as AppliedGiftCertificate } from './AppliedGiftCertificate';
+import { getGiftCertificate } from './giftCertificate.mock';
+
+function roundTo(amount: number, decimals: number): string {
+    const factor = 10 ** decimals;
+
+    return (Math.round(amount * factor) / factor).toFixed(decimals);
+}
+
+describe('AppliedGiftCertificate', () => {
+    const config = getStoreConfig();
+    const giftCertificate = getGiftCertificate();
+
+    it('renders correctly if gift certificate is applied', () => {
+        render(<AppliedGiftCertificate giftCertificate={giftCertificate} />);
+
+        const usedAmount = roundTo(giftCertificate.used * config.shopperCurrency.exchangeRate, parseInt(config.shopperCurrency.decimalPlaces, 10));
+        const remainingAmount = roundTo(giftCertificate.remaining * config.shopperCurrency.exchangeRate, parseInt(config.shopperCurrency.decimalPlaces, 10));
```

Comment:
Could you elaborate on why we should not hardcode static values here?

Reply:
The mock has amount like this https://github.com/bigcommerce/checkout-js/blob/master/packages/core/src/app/giftCertificate/giftCertificate.mock.ts#L6 which after going through currencyFormatter becomes `22.40` since exchange rate is `1.12` with 2 decimal places as part of config.

Reply:
In other words, can we just write something like `expect(screen.getByTestId('giftCertificate-amount')).toHaveTextContent('22.40');` instead?

Reply:
That’s ok as well, but feel its better to have the config and amount. Or else it can get confusing since mock value of 20 becomes 22.40. And if someone in future changes the mock, the test will fail.